libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -92,6 +92,9 @@ void AP_AHRS_NavEKF::update(void)
     update_SITL();
 #endif
 
+    // update AOA and SSA
```

Comment:
Angle of attack is the main parameter for anything that has wings that produce the lift. I can not see any use from it in Copter of Rover.